pages/apple-silicon/how-to/update-os-mac-mini.mdx

```diff
@@ -53,7 +54,9 @@ The recommended method to update the macOS is to reinstall your Mac mini with an
       3. Go back to **System settings > General > Software** update and click the Info button next to **Beta updates** to enable beta versions updates.
     </Message>
 
-### Upgrading the OS using SSH
+    <Message type="important">
+      Your Mac mini may restart during the upgrade process and be unavailable while the software update is installed. Once the update is successfully completed, it will become available again.### Upgrading the OS using SSH
```

Comment:
```suggestion
      Your Mac mini may restart during the upgrade process and be unavailable while the software update is installed. Once the update is successfully completed, it will become available again.
```